packages/material-ui-system/src/createStyled.d.ts

```diff
@@ -188,9 +201,13 @@ export interface CreateMUIStyled<Theme extends object = DefaultTheme> {
     }
   >;
 
-  <C extends React.JSXElementConstructor<React.ComponentProps<C>>>(
+  <
+    C extends React.JSXElementConstructor<React.ComponentProps<C>>,
+    ComponentProps extends {} = any,
```

Comment:
Can we default to `unknown` by default following "strict-by-default"?

Reply:
Setting default value to `unknown` caused an error because of extending `{}` and `''`, so I set default values to `{}` and `''` respectively. Is this okay? or is there workaround to set default value to `unknown`? https://github.com/mui-org/material-ui/pull/27983/commits/e406f0c8d1afcd1e3c45c3af33f5b2d7fdc2fdd6

Reply:
But then, I get these two kinds of type errors everywhere with these default values as well, so should I stick to `any`?
`Property 'root' does not exist on type 'Record<"", CSSInterpolation>'.`
`Property 'color' does not exist on type '{}'.`

Reply:
Because of extending `{}` and `string`, it causes a type error. Is there any workaround for this?

Reply:
If we want to implement "strict-by-default" maybe we can default to {} and ""?

Reply:
@eps1lon what do you think? Is it worth the breaking change? I think it is too late now to introduce those, maybe we can just add the generics to allow developers to make it stricter.